### PR TITLE
Add id corrs

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Comment by pnallin** [comment]
For infomation: I've changed both BBB hostnames (according to the table) a few hours ago!

**Comment by xresende** [comment]
> For infomation: I've changed both BBB hostnames (according to the table) a few hours ago!

nice. I noticed the original host names yesterday and I was going to comment with you. thanks!

**Comment by xresende** [comment]
> @xresende I noticed that the excitation curves are wrong by a factor of approximately 5. We will measure the excitation curves on Monday up to 5A, but for now I suggest using the values ​​from the model (1A): https://github.com/lnls-ima/si-fast-correctors/blob/master/links-official-FC1-fch/analysis.txt

I used arbitrary values (5A -> 30 urad @ 3GeV). I will update the PR with the model values that you pointed. thanks.

File: magnet/excitation-data/si-corrector-ids-ch.txt [hunk 1]
```diff
@@ -8,13 +8,14 @@
 
 # EXCITATION DATA
 # ===============
--005.000  -3.0000e-04 +0.0000e+00
-+005.000  +3.0000e-04 +0.0000e+00
+-001.000  -1.6409e-04 +0.0000e+00
++001.000  +1.6409e-04 +0.0000e+00
```
Comment (fernandohds564) [inline magnet/excitation-data/si-corrector-ids-ch.txt:12]: @xresende, isn't these values for half of the model?

Comment (xresende) [reply magnet/excitation-data/si-corrector-ids-ch.txt:12]: ops! they are indeed for half model. thanks. I'll fix it!

Comment (xresende) [reply magnet/excitation-data/si-corrector-ids-ch.txt]: done.

**Pull request merged**